scripts/deployment/upload_model_to_huggingface.py

```
@@ -0,0 +1,1233 @@
+"""
+🚀 UPLOAD CUSTOM TRAINED MODEL TO HUGGINGFACE HUB
+=================================================
+Upload your custom-trained emotion detection model to HuggingFace Hub 
+so it can be used in production deployment.
+"""
+
+import os
+import sys
+import json
+import shutil
+from pathlib import Path
```

Ruff (0.12.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)